terraform/concourse/elb.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_elb" "concourse" {
   name            = "${var.env}-concourse"
-  subnets         = ["${var.subnet0_id}"]
+  subnets         = ["${split(",", var.infra_subnet_ids)}"]
```

Comment:
Good point. We wanted to make current TF deployment a bit more general. With this config you can place concourse into any subnet without ELB reconfiguration (e.g. in zone failure).

I also think we should use concourse_subnet (like we use bosh_subnet) instead of subnet0 in both TF variable name and in concourse manifest. But that's a bit out of scope of this PR.